atd-vze/src/views/Tools/ToolsUploadNonCR3.js

```diff
@@ -163,7 +163,9 @@ const ToolsUploadNonCR3 = () => {
     }
 
     if (!isValidCoord(record["longitude"], record["latitude"])) {
-      errors.push("Invalid coordinate pair");
+      errors.push(
+        "Invalid coordinate pair (for example, -97.7404, 30.2747 is a valid Austin coordinate pair)"
```

Comment:
Thanks for adding this. An example is always helpful for me when I'm trying to match the form of something.

Reply:
💯